CountingJS/index.css

```
@@ -0,0 +1,3 @@
+body {
+	margin: 0;
```

It looks like your text editor is configured to use hard tabs instead of soft tabs (spaces). It's a more common coding convention to use spaces, so I recommend changing your VS Code settings to use soft tabs with 4 spaces. It will make your indentations look smaller and more consistent with professional standards.

```suggestion
    margin: 0;
```